src/Stripe.net/Stripe.net.csproj

```diff
@@ -50,10 +50,6 @@
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
   </ItemGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' ">
-    <DefineConstants>$(DefineConstants);NET45</DefineConstants>
```

Comment:
Turns out this wasn't needed. Preprocessor constants are automatically defined for the target framework: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-target-frameworks.